spec/spec_helper.rb

```diff
@@ -7,7 +7,6 @@ def ci_build?
 
 require 'simplecov'
 require 'coveralls'
-Coveralls.wear!
 SimpleCov.root(File.expand_path('../..', __FILE__))
 SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
                                                                  Coveralls::SimpleCov::Formatter,
```

Comment:
What is this formatter used for after Coveralls is removed?

Reply:
it seemed better to not change the simplecov formatter just to stop reporting failures.

Reply:
on the other hand, dropping `Coveralls.wear!` doesn't seem to have fixed the issue at all.

Reply:
rebuilding with coveralls dropped entirely.